pkg/aggregate/events_test.go

```diff
@@ -23,21 +23,16 @@ func (e Event) Apply(a Aggregate) error {
 }
 
 func TestNewEventsQueue(t *testing.T) {
-	agg := Aggregate{}
-
 	event1 := Event{ID: 1}
 	event2 := Event{ID: 2}
 
-	es := aggregate.NewEventsQueue(agg)
+	es := aggregate.EventsQueue[Aggregate]{}
```

Comment:
nice!
I love it that the empty value is a valid value, no need for a constructor <3

Reply:
❤️